packages/react/src/components/groups/groups.test.tsx

```diff
@@ -43,6 +44,48 @@ describe("Groups", () => {
     expect(screen.queryByText("Test")).not.toBeInTheDocument();
   });
 
+  test("should render children when the SDK is ready and the user belongs to the specified groups, as string", () => {
+    const userWithGroups = createTestUser()
+    const groups = Array
+      .from(Array(faker.number.int({ min: 3, max: 10 })))
+      .map(() => faker.company.buzzNoun())
+
+    userWithGroups.getGroups = vi.fn(() => groups);
+
+    const [belongsTo] = faker.helpers.shuffle(groups)
```

Comment:
What is the added value of using faker here vs a predefined set of constants? Personally I find this harder to read so the live documentation aspect of tests is a bit hurt. Is it a good trade off?

Reply:
When we use static data we risk introducing false positives in tests.

Here's an example scenario:
1. The group `groupsa` is used to write a test when the `<Groups>` component is created, it's carefully double checked to assure it fails when the value is changed.
2. The test gives us peace of mind when making changes because it should guarantee this particular feature works. It's tested.
3. We change `<Groups>` some time later, accidentally we introduce an edge case where if exactly `groupsa` is provided as the prop, this test _always_ passes but otherwise the feature is actually broken. We've introduced a magic string basically.
4. We assume the component works because the test has always passed and is still passing, it must be healthy. Publish.
5. 💥

By introducing some randomness we prevent these kind of magic values and increase the robustness of the test, without it we're testing that specific values work as opposed to testing that the data flows correctly.

Trade off: we lose some readability, agree!

Reply:
Hmm I think we could have both, right? Do you think it would be too redundant to add a more streamlined test with a constant just for the live docs purpose? With that we solved both problems 😃

Reply:
It would essentially be the same, just without the iteration and generating group names. So a shorter, more readable version. I don't think it's too important though given that it's not a complex component/test, please proceed in any way you think makes sense!

Reply:
Lets do both, I think it makes sense